packaging/dbscripts/upgrade/pre_upgrade/0000_config.sql

```diff
@@ -841,6 +841,8 @@ select fn_db_add_config_value_for_versions_up_to('PredefinedVMProperties', 'sap_
 select fn_db_add_config_value('HostMonitoringWatchdogIntervalInSeconds','900','general');
 select fn_db_add_config_value('HostMonitoringWatchdogInactivityThresholdInSeconds','1200','general');
 
+select fn_db_add_config_value('AllowedHostDeviceDrivers','vfio-pci,pci-stub','general');
```

Comment:
please add it also to engine-config - in this case, not because I think it is likely to change by users but to have a proper documentation for this config value